itou/templates/apply/includes/list_table.html

```diff
@@ -14,13 +14,13 @@
                 {% include 'common/tables/th_with_sort.html' with order=order ascending_value=order.JOB_SEEKER_FULL_NAME_ASC name="Candidature de" extra_button_attributes='data-emplois-elements-visibility-on-selection="disabled"' only %}
             {% endif %}
             {% include 'common/tables/th_with_sort.html' with order=order ascending_value=order.CREATED_AT_ASC name="Émise le" extra_button_attributes='data-emplois-elements-visibility-on-selection="disabled"' only %}
-            <th scope="col">Statut</th>
             {% if job_applications_list_kind is JobApplicationsListKind.SENT %}<th scope="col">Employeur</th>{% endif %}
             <th scope="col">Poste</th>
             <th scope="col">Emetteur</th>
```

Comment:
nit: présent depuis #5242 mais j'aurais bien mis un accent ici (c'est la diff du snapshot qui me l'a révélée)

```suggestion
            <th scope="col">Émetteur</th>
```